src/templates/csr/capi-multiple-hosted/index.svelte

```diff
@@ -0,0 +1,130 @@
+<script context="module" lang="ts">
+	import '$templates/components/fonts/Sans.css';
+	import '$templates/components/fonts/SansBold.css';
+</script>
+
+<script lang="ts">
+	import {
+		isValidReplacedVariable,
+		type GAMVariable,
+		addTrackingPixel,
+	} from '$lib/gam.js';
+	import CapiHostedCard from '$templates/components/CapiHostedCard.svelte';
+	import HostedHeader from '$templates/components/HostedHeader.svelte';
+	import Resizer from '$templates/components/Resizer.svelte';
+	import { addOverridesToCardData, retrieveCapiData } from '$lib/capiMultiple';
+	import type { CapiCardOverride } from '$lib/types/capi';
+
+	export let SeriesURL: GAMVariable;
+	export let BrandLogo: GAMVariable;
+	export let BrandColour: GAMVariable;
+	export let TrackingId: GAMVariable;
+	export let Article1Headline: GAMVariable;
+	export let Article1Image: GAMVariable;
+	export let Article1URL: GAMVariable;
+	export let Article2Headline: GAMVariable;
+	export let Article2Image: GAMVariable;
+	export let Article2URL: GAMVariable;
+	export let Article3Headline: GAMVariable;
+	export let Article3Image: GAMVariable;
+	export let Article3URL: GAMVariable;
+	export let Article4Headline: GAMVariable;
+	export let Article4Image: GAMVariable;
+	export let Article4URL: GAMVariable;
+
+	let cardOverrides: CapiCardOverride[] = [
+		{
+			headline: Article1Headline,
+			image: Article1Image,
+			url: Article1URL,
+		},
+		{
+			headline: Article2Headline,
+			image: Article2Image,
+			url: Article2URL,
+		},
+		{
+			headline: Article3Headline,
+			image: Article3Image,
+			url: Article3URL,
+		},
+		{
+			headline: Article4Headline,
+			image: Article4Image,
+			url: Article4URL,
+		},
+	];
+
+	const getCards = retrieveCapiData(SeriesURL, cardOverrides).then((response) =>
+		addOverridesToCardData(response.articles, cardOverrides, BrandLogo),
+	);
+
+	if (isValidReplacedVariable(TrackingId)) addTrackingPixel(TrackingId);
+
+	let height: number = -1;
+</script>
+
+<aside bind:clientHeight={height} style="--brand-colour: {BrandColour}">
+	{#await getCards}
+		<h3>Loading Content...</h3>
+	{:then multiple}
+		<HostedHeader logo={multiple.logo} />
+		<div class="cards-container">
+			{#each multiple.cards as card}
+				<CapiHostedCard {card} />
+			{/each}
+		</div>
+	{:catch}
+		<h3>An error occurred whilst loading content</h3>
+	{/await}
+</aside>
+<Resizer {height} />
+
+<style lang="scss">
+	aside {
```

Comment:
It looks like we're missing a border along the top of the ad - it should be the same colour as the cards on the ad
<img width="443" alt="Screenshot 2024-01-11 at 10 38 12" src="https://github.com/guardian/commercial-templates/assets/108270776/f285a34b-609b-4680-8f70-9104facb33c5">